plugins/easyimage/plugin.js

```diff
@@ -204,7 +204,8 @@
 							var contextView = editor._.easyImageToolbarContext.toolbar._view;
 
 							if ( contextView.rect.visible ) {
-								contextView.attach( contextView._pointedElement );
+								// We have to disable focusing balloon toolbar to prevent loosing focus by an image (https://github.com/ckeditor/ckeditor-dev/issues/1529).
```

Comment:
You can reference GitHub issues using a short notation, so here `#1529` would be enough. See other references like that across the project.